pkg/runtime/channels/channels.go

```diff
@@ -198,6 +198,19 @@ func (c *Channels) AppHTTPEndpoint() string {
 	}
 }
 
+func (c *Channels) AppHealthEndpoint() string {
```

Comment:
Do we have app health endpoint for gRPC? If not, it is ok to handle only http endpoint for now.

Reply:
Yes handling http now